jinja2/includes/newsletter.html

```diff
@@ -1,31 +1,30 @@
 <div class="newsletter">
     <form id="newsletterForm" class="newsletter-form nodisable" name="newsletter-form" action="https://www.mozilla.org/en-US/newsletter/" method="post">
         <div class="newsletter-head">
-            <h2 class="newsletter-teaser">{{ _('Learn the best of web development') }}</h2>
-            <p class="newsletter-description">{{ _('Get the latest and greatest from MDN delivered straight to your inbox.') }}</p>
+            <h2 class="newsletter-teaser">{{ _('Learn the best of web development. Get the latest and greatest from MDN delivered to your inbox.') }}</h2>
+            <div id="newsletterPrivacy" class="form-group form-group-agree newsletter-group-privacy">
+                <input type="checkbox" id="newsletterPrivacyInput" name="privacy" required>
+                <label for="newsletterPrivacyInput">
```

Comment:
I have seen it be pretty standard practice to nest the `checkbox` input inside the `label` tag. What do you think? If you do decide to do it here, then we should do it in the `jsx` as well.

Reply:
I noticed that on mobile the alignment of the checkbox with the text is a little off:

![Screenshot 2020-02-18 at 09 37 36](https://user-images.githubusercontent.com/10350960/74714034-6c7a0580-5232-11ea-8236-10192729e8c6.png)

Moving the `checkbox` inside the `label` resolves that problem so, let's make that change.